extension/src/cli/index.ts

```diff
@@ -101,6 +96,60 @@ export class Cli extends Disposable implements ICli {
     }
   }
 
+  protected createBackgroundProcess(options: ProcessOptions) {
+    const { baseEvent, stopWatch } = this.getProcessDetails(options)
+    try {
+      const backgroundProcess = this.createProcess(baseEvent, {
+        detached: true,
+        ...options
+      })
+
+      return new Promise<string>(resolve => {
```

Comment:
[F] Here we are attaching a listener to the readable stream so that we can get the initial stdout then cut the process loose.